pkg/scripts/os_test.go

```diff
@@ -572,7 +581,12 @@ func TestUpgradeKubeadmAndCNIAmazonLinux(t *testing.T) {
 func TestUpgradeKubeadmAndCNIFlatcar(t *testing.T) {
```

Comment:
Is it okay to extend these 2 test cases or should I create extra ones?

Reply:
I think it's fine